arm/Microsoft.Synapse/workspaces/.bicep/nested_cmkRbac.bicep

```diff
@@ -0,0 +1,39 @@
+param keyvaultName string
+param workspaceIdentity string
+param usesRbacAuthorization bool = false
+
+// Workspace encryption - Assign Workspace System Identity Keyvault Crypto Reader at Encryption Keyvault
+resource keyVault 'Microsoft.KeyVault/vaults@2019-09-01' existing = {
+  name: keyvaultName
+}
+
+// Assign RBAC role Key Vault Crypto User
+resource workspace_cmk_rbac 'Microsoft.Authorization/roleAssignments@2021-04-01-preview' = if (usesRbacAuthorization) {
+  name: guid(deployment().name)
```

Comment:
This will not be idempotent. With an IaC module, it should for every deployment control the same initial resource we deploy. So think there are some needed changes here to get the rbac set with the module and controlled going forward.